docs/MapColonies/Raster/services/catalog/profile_v2_0.md

```diff
@@ -0,0 +1,34 @@
+---
+id: raster-profile-v2
+slug: raster-catalog-profile-v2
+description: v2 Profile
+title: Profile v2
+tags:
+  - raster
+  - catalog
+  - v2
+---
+
+# Raster Catalog Profile Information - v2
+
+1. **typename** = `mc:MCRasterRecord`
+2. **main_namespace** = `http://schema.mapcolonies.com/raster` (This site doesn't exist)
+3. The **Raster** sub-system Catalog profile fields with information of each of them:
+
+
+
+:::tip
+<p style={{color:"red", }}><b>Important: New version is comming soon!</b></p>
+:::
+
+## Coming new parameters for v2
+
+| **PYCSW Queryable/XML <br/> Element Name** | **Type** | **Description** |
```

Comment:
Please add `valid values` column to the table (for `v1` as well).

Reply:
i've already added example with valid response in the bottom of v1 page, i don't want to add more data to the table as it will be messy